### ISSUE TITLE
Where is the specification?

### ISSUE THREAD

**Issue by fingolfin** [ebkalderon/libnar#4]
The README says:
> Fully specified, no undefined or implementation-specific behavior

But I can't a specification in this repository, nor a link to one?

**Comment by ebkalderon** [comment]
Good point! I could definitely add a link to the README. The specification for the NAR format can be found on page 92-94 of the Nix PhD thesis ([phd-thesis.pdf](https://edolstra.github.io/pubs/phd-thesis.pdf)).
